composer.json

```
@@ -13,10 +13,10 @@
     ],
     "require": {
         "php": ">=7.0",
-        "guzzlehttp/guzzle": "^6.3",
-        "illuminate/events": "~5.5 || ~6.0 || ~7.0",
-        "illuminate/notifications": "~5.5 || ~6.0 || ~7.0",
-        "illuminate/support": "~5.5 || ~6.0 || ~7.0"
+        "guzzlehttp/guzzle": "^6.3 || ^7.0.1",
```

can you do this sorry (allow minor versions above 7.0)
```suggestion
        "guzzlehttp/guzzle": "^6.3 || ^7.0",
```